app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -9,13 +9,11 @@ import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
-
```

Comment:
Suggestion 1: Removed unused commented-out code to prevent the code from becoming unnecessarily large.
Source: https://rules.sonarsource.com/kotlin/RSPEC-125 and https://docs.sonarqube.org/display/SONARQUBE45/Not+Enough+or+Too+Many+Comments

Reply:
Yes, definitely.  Good call.